staking_deposit/deposit.py

```diff
@@ -53,8 +54,10 @@ def cli(ctx: click.Context, language: str, non_interactive: bool) -> None:
 
 cli.add_command(existing_mnemonic)
 cli.add_command(new_mnemonic)
+cli.add_command(generate_bls_to_execution_change)
 
 
 if __name__ == '__main__':
     check_python_version()
+    print('\n***Using the tool on an offline and secure device is highly recommended to keep your mnemonic safe.***\n')
```

Comment:
@james-prysm, per https://github.com/ethereum/staking-deposit-cli/pull/313/files#r1087833602, I made a warning at the beginning.

Hey @CarlBeek It is printed before we select language now because I'm not sure how to print/echo the message before the user input sensitive data. do you have an idea of how to make it translatable?

Reply:
I don't see an obvious way of doing so, unfortunately